test/integration/connect/envoy/Dockerfile-socat.windows

```diff
@@ -0,0 +1,11 @@
+
+FROM mcr.microsoft.com/windows/servercore:1809
+
+RUN mkdir "C:\\socat"
+WORKDIR "C:\\socat"
+
+COPY ./socat-1.7.3.0-windows .
+
+#Add socat to PATH
+ENV PATH C:\\socat;%PATH%
+RUN socat.exe -V
```

Comment:
Could we use CMD command to execute the socat.exe instead of creating an environment variable ENV and then execute the RUN command?

E.g. `CMD ["socat-path/socat.exe", "-v"]`

Reply:
is the socat.exe dynamically built? my doubt is because I don't see the .exe file